### DISCUSSION TITLE
unable to access InvenTree from public Internet

### DISCUSSION THREAD

**Discussion by szmolnar** [inventree/InvenTree#7887] [Q&A]
Hi!

I would need some help on how to properly set up Caddy.
I successfully installed InvenTree on docker, and everything is working just fine on my local network but only on HTTP.

Port forwarding on my router is just fine.
I checked it by creating a simple Caddyfile replacing the original one provided on InvenTree github page. 

**My test Caddyfile:**

http://192.168.5.168:80, http://mydomain.org:80, https://mydomain.org:443

#respond "Hello world!"
#reverse_proxy https://mydomain.org:443 inventree-server:8000

reverse_proxy {$INVENTREE_SERVER:"http://inventree-server:8000"}

With the above simple Caddyfile InvenTree is accessible from the internet, but has some issues on login etc.
The original Caddyfile modified by me (below) is not working, my server is not reachable from the internet.

**My modified Caddyfile:**

(log_common) {
        log {
                output file /var/log/caddy/{args[0]}.access.log
        }
}

(cors-headers) {
        header Allow GET,HEAD,OPTIONS
        header Access-Control-Allow-Origin *
        header Access-Control-Allow-Methods GET,HEAD,OPTIONS
        header Access-Control-Allow-Headers Authorization,Content-Type,User-Agent

        @cors_preflight{args[0]} method OPTIONS

        handle @cors_preflight{args[0]} {
                respond "" 204
        }
}

{$INVENTREE_SITE_URL:https://mydomain.org:443} {
        import log_common inventree

        encode gzip

        request_body {
                max_size 100MB
        }

        handle_path /static/* {
                import cors-headers static

                root * /var/www/static
                file_server
        }

        handle_path /media/* {
                import cors-headers media

                root * /var/www/media
                file_server

                header Content-Disposition attachment

                forward_auth {$INVENTREE_SERVER:"http://inventree-server:8000"} {
                        uri /auth/
                }
        }

        reverse_proxy {$INVENTREE_SERVER:"http://inventree-server:8000"}
}
 
Any help is welcome!

Many thanks!

**Comment by szmolnar** [comment]
Well, I solved it by modifying the env variable INVENTREE_SITE_URL only to my public https address.
Information found inside the caddy file was misleading, I thought I need to modify it there.

**Reply by matmair** [reply]
https://docs.inventree.org/en/latest/start/config/#server-access
